src/components/expanded-state/custom-gas/FeesPanel.js

```diff
@@ -301,11 +301,11 @@ export default function FeesPanel({
         gweiToWei(newGweiMaxPriorityFeePerGas)
       );
 
-      if (newMaxPriorityFeePerGas.amount < 0) return;
+      if (greaterThan(0, newMaxPriorityFeePerGas.amount)) return;
```

Comment:
greaterThanOrEqualTo assuming 0 is an invalid value

Reply:
yes is invalid so I added the `Invalid fee` label to the buttons when this happens. For this specific piece code we need to allow 0 because the user could erase everything from the input, having an empty string, if we don't allow this we won't know about these empty string cases and won't be able to tell the user the fee is invalid